setup.py

```diff
@@ -134,6 +134,7 @@ def find_package_data(where='.', package='',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
```

Comment:
This line (135: `'Programming Language :: Python',`) should be deleted, so PyPi won't think it's python 2 compatible.

Reply:
Just tried it and you are right, the line with the `:: Only` is the game changer.

Reply:
I don't think that we need to remove this line. at least we have something like this in Django:
https://github.com/django/django/blob/6410d38ca7255961c08f1d657fdf920555d113f3/setup.cfg#L17

Reply:
Thats what I ment, there is no need to delete that line. 
All fine I guess, so go for it 👍